public/assets/css/monopoly.css

```diff
@@ -73,3 +77,17 @@
 .racecar {
     background-color: #999900;
 }
+
+.player-info > .property-container {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.player-property {
+    font-size: 9px;
```

Comment:
So the thing that confused me is that 1em is the current font size. So changing the font size based on ems felt strange...